arcs/crowdflower.py

```diff
@@ -46,11 +47,13 @@ def create_job_from_copy(api_key=None, job_id=None):
         job_id (int): The unique job identifier of the job to copy
 
     Returns:
-        A new Arcs Job
+        A new Job
     """
-    job_id = job_id or 844991
+    job_id = job_id or 911946
     api_key = api_key or os.environ["CROWDFLOWER_API_KEY"]
-    url = "https://api.crowdflower.com/v1/jobs/{}/copy.json?key={}&gold=true".format(job_id, api_key)
+    url = "https://api.crowdflower.com/v1/jobs/{}/copy.json?key={}&gold=true".format(
```

Comment:
Generally I feel like we should be passing params as a `params` dict, but this is not necessary for this commit.